### PR TITLE
41 4.16 application confirmation

### PR DESCRIPTION
**Pull request by Raandom**

### CODE REVIEW

File: config/locales/de.application_letters.yml [hunk 1]
```diff
@@ -73,3 +73,11 @@ de:
...
     pending_after_deadline: "In Bearbeitung"
     alternative: "Nachrücker"
+
+
+  controllers:
+    application_letters:
+      confirmation_mail:
+        sender: "workshop@hpi.de"
+        subject: "Bewerbung erhalten"
+        content: "Hallo, \n\rDu hast dich erfolgreich fuer  {{semimar_name}} beworben. Deine Bewerbung wird jetzt bearbeitet. Fuer weitere Information besuche das Workshop Portal. Viele Grueße, das HPI Workshop Team"
```
Comment (c0rnyc0rny) [inline config/locales/de.application_letters.yml]: If possible use "ü" instead of "ue"

**Review by c0rnyc0rny — CHANGES_REQUESTED** [review]

**Review by c0rnyc0rny — APPROVED** [review]

File: config/locales/de.application_letters.yml [hunk 2]
```diff
@@ -73,3 +73,11 @@ de:
     pending_before_deadline: "Beworben"
     pending_after_deadline: "In Bearbeitung"
     alternative: "Nachrücker"
+
+
+  controllers:
+    application_letters:
+      confirmation_mail:
+        sender: "workshop@hpi.de"
```
Comment (bjrne) [inline config/locales/de.application_letters.yml:81]: I would have liked to see the usage of the default value or any other user changeable email here

File: config/locales/de.application_letters.yml [hunk 3]
```diff
@@ -73,3 +73,11 @@ de:
...
+        subject: "Bewerbung erhalten"
+        content: "Hallo, \n\rDu hast dich erfolgreich für  {{semimar_name}} beworben. Deine Bewerbung wird jetzt bearbeitet. Für weitere Information besuche das Workshop Portal. Viele Grüße, das HPI Workshop Team"
```
Comment (bjrne) [inline config/locales/de.application_letters.yml:83]: `semimar_name`

**Comment by bjrne** [comment]
@Raandom @h0rnyc0rny

**Comment by cmfcmf** [comment]
This has broken stuff and cost me quite some time.. Part of it was fixed by #490. Another issue is #492.

**Comment by bjrne** [comment]
@Raandom @h0rnyc0rny It would be nice if you'd actually look at your mentions. You can unsubscribe from the general notifications, the level should then lower to an acceptable one. Or you could visit https://github.com/issues/mentioned and https://github.com/pulls/mentioned a bit more. Please.

**Comment by Raandom** [comment]
This didnt break anything That wasnt Broken Before. Fixing the typo tomorrow.

**Pull request merged**